camera_1:
  parent_tf: box_bundle_1
  xyz: "3.95527 2.43867 1.0147"
  rpy: "1.54495 -0.30657 -2.81921"
  xyz_std: "0.01282 0.00895 0.00728"
  rpy_std: "0.00155 0.00127 0.00233"
  samples: 16
camera_2:
  parent_tf: box_bundle_4
  xyz: "0.44337 2.58421 4.85651"
  rpy: "-2.51973 1.24145 -1.08373"
  xyz_std: "0.00404 0.00114 0.00174"
  rpy_std: "0.00193 0.00044 0.00206"
  samples: 13
camera_4:
  parent_tf: box_bundle_1
  xyz: "-3.17345 2.49291 5.40532"
  rpy: "-2.21317 0.87274 -0.81108"
  xyz_std: "0.0015 0.00257 0.00267"
  rpy_std: "0.00047 0.0004 0.00059"
  samples: 14
camera_5:
  parent_tf: box_bundle_1
  xyz: "1.29859 2.37788 4.78913"
  rpy: "2.2297 1.06485 -2.381"
  xyz_std: "0.00239 0.00168 0.00198"
  rpy_std: "0.00081 0.00047 0.0008"
  samples: 15
camera_6:
  parent_tf: box_bundle_4
  xyz: "3.87441 2.42939 8.16913"
  rpy: "2.62979 1.14674 -2.11231"
  xyz_std: "0.00557 0.01103 0.00941"
  rpy_std: "0.0019 0.00132 0.00189"
  samples: 15
camera_3:
  parent_tf: box_bundle_4
  xyz: "3.87441 2.42939 8.16913"
  rpy: "2.62979 1.14674 -2.11231"
  xyz_std: "0.00557 0.01103 0.00941"
  rpy_std: "0.0019 0.00132 0.00189"
  samples: 15
